components/Premium/Content.tsx

```diff
@@ -20,24 +11,32 @@ import {
 	Stripe
 } from '@/app/premium/_components/icons'
 import {
-	Select,
-	SelectContent,
-	SelectItem,
-	SelectTrigger,
-	SelectValue
-} from '@/components/ui/select'
+	Dialog,
+	DialogContent,
+	DialogDescription,
+	DialogHeader,
+	DialogTitle,
+	DialogTrigger
+} from '@/components/ui/dialog'
+import {
+	Command,
+	CommandEmpty,
+	CommandGroup,
+	CommandInput,
+	CommandItem,
+	CommandList
+} from '@/components/ui/command'
 import { Avatar, AvatarFallback, AvatarImage } from '@/components/ui/avatar'
 import PlansComparison from '@/app/premium/_components/PlansComparison'
 import CheckoutButton from '@/app/premium/_components/checkoutButton'
+import type { PricingData, DiscordGuild } from '@/app/premium/_types'
 import DiscordLoginButton from '@/components/DiscordLoginButton'
-import { PricingData, DiscordGuild } from '@/app/premium/_types'
-import { ServersListSkeleton } from '@/app/premium/_components'
 import { CheckIcon, Link } from 'lucide-react'
 import { useIdToken } from '@/helpers/hooks'
-import { Suspense, useState } from 'react'
+import { Fragment, useState } from 'react'
```

Comment:
We can remove this Fragment import based on removals below.
```suggestion
import { useState } from 'react'
```